autopush/web/base.py

```diff
@@ -16,6 +16,7 @@
 )
 from autopush.exceptions import InvalidRequest
 from autopush.router.interface import RouterException
+from autopush.endpoint import DEFAULT_ERR_URL
```

Comment:
I'd rather endpoint import this from web/base than vice-versa, since endpoint.py is the one slated for the chopping block.